src/main/telemetry/crsf.c

```diff
@@ -282,6 +296,20 @@ void crsfFrameHeartbeat(sbuf_t *dst)
     sbufWriteU16BigEndian(dst, CRSF_ADDRESS_FLIGHT_CONTROLLER);
 }
 
+/*
+0x28 Ping
+Payload:
+int8_t    origin_add ( Destination Device address )
+int8_t    origin_add ( Origin Device address )
```

Comment:
still needs fixing 🙂

Reply:
Fixed. Apologies for being so slow on this!